libs/SalesforceHybridSDK/SalesforceHybridSDK.xcodeproj/xcshareddata/xcschemes/SalesforceHybridSDKStatic.xcscheme

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0710"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "NO">
+      <BuildActionEntries>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "CEA884421C191795008D871B"
+               BuildableName = "libSalesforceHybridSDKStatic.a"
```

Comment:
Exactly.  I think that's why Xcode changed them for me.  If you play around in the Schemes editor a little bit, it'll probably change them for you automatically. :)